src/main/python/hydra/lib/runtestbase.py

```diff
@@ -1,643 +1,11 @@
-__author__ = 'sushil'
```

Comment:
I did a "git mv runtestbase.py hydrabase.py" But still git shows them as two different files. 
There are no changes between the two files.